vimeo-deeplink/build.gradle

```diff
@@ -26,14 +26,14 @@ ext {
 }
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.3"
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 23
-        versionCode 2
-        versionName "1.1.0"
+        targetSdkVersion 25
+        versionCode 3
+        versionName "1.2.0"
```

Comment:
I think this line should be
```groovy
versionName libraryVersion
```
so that we ensure that we never have a different library version from the version name we use. Also, we should probably create a `libraryCode` constant next to the `libraryVersion` constant that contains the versionCode